.github/workflows/deploy-sandbox.yaml

```diff
@@ -72,7 +72,7 @@ jobs:
     runs-on: ubuntu-latest
     needs: [variables, deploy]
     permissions:
-      issues: write
+      pull-requests: write
```

Comment:
Can you elaborate on why you chose `pull-requests` and why `issues` doesn't work? I'm looking at [this Github documentation](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token) and not as familiar with workflow stuff

Reply:
So I'm not totally clear on why this works, the documentation says that *either* should be sufficient. The proof is just that the job works on this PR and does not on others. My *guess* is that because the comment gets made on the PR and not on the issue, it needs PR permissions, but IDK why they didn't just make that a separate endpoint.

Reply:
Question from that -- is there a reason why we don't just do both `pull-request` and `issues` like in the example?

Reply:
We can, we just might be adding slightly more permissions than necessary which would be a (very small) violation of the principle of least privilege. It's an easy fix if needed.